RELEASE.md

```diff
@@ -52,4 +55,4 @@ The following steps should be always run, regardless of whether an automated or
    1. Mark this version of Stencil as 'released' in JIRA on the 'Releases' page.
    2. Move the task card in this current sprint to the 'Done' swim-lane.
    3. Stub out the next release and task for the release in JIRA.
-5. Ensure all GitHub Issues associated with stories/tasks that shipped in this version of Stencil are closed.
+5. Ensure all GitHub Issues associated with stories/tasks that shipped in this version of Stencil are closed.
```

Comment:
A few more things we forgot to do yesterday 🤦 
```suggestion
5. Ensure all GitHub Issues associated with stories/tasks that shipped in this version of Stencil are closed.
6. Let folks in Ionic know about the release in the #whathappened Slack channel with a link to the changelog
7. End the code freeze in the Stencil team Slack channel
```

Reply:
I thought we weren't going to be doing announcements in #whathappened for patch releases since they'll be a weekly thing?